src/quo2/components/inputs/input/view.cljs

```diff
@@ -188,3 +188,5 @@
     (if (= type :password)
       [password-input base-props]
       [base-input base-props])))
+
+(def input (quo.theme/with-theme input-internal))
```

Comment:
When we wrap the internal component with HOCs or export components from the quo2.core namespace, the doc string is not being forwarded to the newly defined function. I'm curious if there is a way to fix this.

`internal-input` has a doc-string | But `input` doesn't
-|-
<img width="1002" alt="Screenshot 2023-07-25 at 09 53 57" src="https://github.com/status-im/status-mobile/assets/19279756/95eef3eb-4170-4e5a-8e78-1ea48c1b3592"> | <img width="656" alt="Screenshot 2023-07-25 at 09 54 05" src="https://github.com/status-im/status-mobile/assets/19279756/49802c65-eaf7-4a14-922b-5f7e169a72cf">


(Not in the scope of this PR)

Reply:
Yep @ibrkhalil pointed out you add docstring on the outer layer and it works. Perhaps someone can create a follow up issue to move the docstrings, and maybe even clean a few up while they're doing it :)

Reply:
Even if we try that approach when we define the components in `quo2.core` like this`(def input quo2.components.inputs.input.view/input)`, the doc string is not being preserved.

Reply:
since the docstring is [metadata](https://clojure.org/reference/metadata), it might be possible to forward it with functions like `meta`, `with-meta`, etc.

Reply:
@erikseppanen @J-Son89 @ajayesivan 

Even if we add it as metadata, some IDEs/Editors (like IntelliJ) that perform static code analysis will not read the docstring.